app/controllers/taxonomy_signups_controller.rb

```diff
@@ -45,6 +47,10 @@ def load_taxon
       .content_item(taxon_path)
   end
 
+  def validate_taxon_document_type
+    redirect_to '/' and return unless @taxon['document_type'] == 'taxon'
```

Comment:
It shouldn't be `nil`, I think we'll always have a content item by the time we get here - 404s from content store raise exceptions which somehow magically turn into a 404 response from email-alert-frontend, and any other 4xx or 5xx response from content store should also raise an exception before we get this far.